### PR TITLE
Add CMakeLists for Graphics Gems IV and fix all errors on VS2015.

### PR DESCRIPTION
**Pull request by gonsolo**

### CODE REVIEW

File: gemsiv/delaunay/quadedge.C [hunk 1]
```diff
@@ -1,4 +1,9 @@
 #include "quadedge.h"
+#include "../../dummy.h"
```
Comment (erich666) [inline gemsiv/delaunay/quadedge.C]: This include may be needed, but seems a bit odd. Could you explain "dummy.h" and what it's for? It looks a bit catch-all.

Comment (gonsolo) [inline gemsiv/delaunay/quadedge.C]: The code in question seems to be using IRIS GL which is not available. I tried to have it compiled by adding all the missing functions. Maybe I should rename dummy.[hc] to irisgl, fakegl or something like that. Do you have a suggestion?

Comment (erich666) [inline gemsiv/delaunay/quadedge.C]: Ah, thanks. Yes, I like fakegl, and please add a comment at the top explaining why this file was added. And thanks very much for bringing this code into this century!

Comment (gonsolo) [inline gemsiv/delaunay/quadedge.C]: I renamed it to fakeirisgl to make it clear which API was used and documented it in fakeirisgl.h.

**Comment by erich666** [comment]
Thanks, this looks good, I appreciate it.

**Pull request merged**